chain/index/api.go

```diff
@@ -57,9 +57,10 @@ func (si *SqliteIndexer) getTipsetCountsAtHeight(ctx context.Context, height abi
 }
 
 func (si *SqliteIndexer) ChainValidateIndex(ctx context.Context, epoch abi.ChainEpoch, backfill bool) (*types.IndexValidation, error) {
-	if !si.started {
-		return nil, xerrors.Errorf("ChainValidateIndex can only be called after the indexer has been started")
+	if !si.started && si.isClosed() {
+		return nil, xerrors.Errorf("ChainValidateIndex can only be called after the indexer has been started and not closed")
```

Comment:
have different if statements for both conditions so the error message can be more granular.

Reply:
Done.